src/rest_framework_jwt/blacklist/models.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.db import models
+
+
+class BlacklistedTokenManager(models.Manager):
+    def delete_stale_tokens(self):
+        return self.filter(expires_at__lt=datetime.utcnow()).delete()
+
+
+class BlacklistedToken(models.Model):
+    token = models.TextField(db_index=True, unique=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, related_name='owner')
```

Comment:
Blacklisted tokens are accessed via `user.owner`?

Reply:
Removed the `related_name` part.